Source/WarpX.cpp

```diff
@@ -3067,6 +3108,9 @@ WarpX::UpperCorner(const Box& bx, const int lev, const amrex::Real time_shift_de
 #elif defined(WARPX_DIM_XZ) || defined(WARPX_DIM_RZ)
     return { grid_max[0] + galilean_shift[0], std::numeric_limits<Real>::max(), grid_max[1] + galilean_shift[1] };
 
+#elif defined(WARPX_DIM_RCYLINDER) || defined(WARPX_DIM_RSPHERE)
+    return { grid_max[0] + galilean_shift[0], std::numeric_limits<Real>::max(), std::numeric_limits<Real>::max() };
```

Comment:
Should we raise an exception here if `galilean_shift[0]` is different than 0?
```suggestion
    return { grid_max[0], std::numeric_limits<Real>::max(), std::numeric_limits<Real>::max() };
```

Reply:
Fixed. Now raises an error if boosted frame is specified.